packages/babel-plugin-proposal-function-bind/src/index.ts

```diff
@@ -14,11 +14,18 @@ export default declare(api => {
     return scope.path.setData("functionBind", id);
   }
 
+  function getObject(bind: t.BindExpression) {
+    if (t.isExpression(bind.object)) {
+      return bind.object;
+    }
+
+    if (t.isMemberExpression(bind.callee)) {
+      return bind.callee.object;
+    }
```

Comment:
We can use an assertion, so that we know that we always fall in one of these two branches.

```suggestion
    t.assertMemberExpression(bind.callee);
    return bind.callee.object;
```

Reply:
Yes, it would be better!